cmd/install/assets/cluster-api/cluster.x-k8s.io_clusters.yaml

```diff
@@ -20,390 +20,6 @@ spec:
     singular: cluster
   scope: Namespaced
   versions:
-  - additionalPrinterColumns:
```

Comment:
where are the changes to this core CRDs coming from?

Reply:
This one got updated part of `make verify` command, not sure why it is removing `v1alpha4` reference, maybe got removed because we removed the `v1alpha4` reference(just a wild guess)